src/screens/MediatorScreen.js

```diff
@@ -70,8 +70,12 @@ const Mediator = (props) => {
             //3- create OOB URL
             const ooburl = await generateOOBURL(myDid)
             console.log("OOB URL: "+ ooburl)
-            // QR Code too big. Need short url
-            setQRCode(ooburl)
+            // SHorten URL
+            const shortQR = await shortenURLRequest(didToMediator, mediatorDID, ooburl)
```

Comment:
@rodolfomiranda need to supply validityInSeconds.... or in the function signature, give it a default

Reply:
you are right. It probably are generating a code with 0 seconds of validity :)
I couldn't add it because it's merged. In next merge add a validity in seconds, such as:
`const shortQR = await shortenURLRequest(didToMediator, mediatorDID, ooburl, 24*60*60)`

Pending a good refactoring to adopt best TS practices...

Reply:
Yes I didn't discover it until after I accepted. No rush, next PR is good.